pkg/infrastructure/gcp/clusterapi/network.go

```diff
@@ -91,3 +91,39 @@ func createInternalLBAddress(ctx context.Context, in clusterapi.InfraReadyInput)
 
 	return ipAddress, nil
 }
+
+// createFirewallRules creates the rules needed for master and worker nodes.
```

Comment:
nit: Maybe a note about the addition of the tag and why. Either here or below. This would be helpful if we need to update or remove it later.

Reply:
I changed this around a bit to create the rules for the workers instead of just adding the tag to the rule created by CAPG.